tests/avify4mtest.c

```diff
@@ -0,0 +1,198 @@
+// Copyright 2022 Google LLC. All rights reserved.
+// SPDX-License-Identifier: BSD-2-Clause
+
+#include "apps/shared/y4m.h"
+#include "avif/avif.h"
```

Comment:
Please change these two lines to:

```
#include "avif/avif.h"

#include "y4m.h"
```

This is how apps/avifdec.c and apps/avifenc.c include y4m.h.

Reply:
Done in #846